api/enterprise-api-ref/users-rbac.markdown

```diff
@@ -367,3 +367,30 @@ API call allowed only for administrator.
 
 Remove role definition.
 API call allowed only for administrator.
+
+## Unlock user
+
+If a system enforces using two-factor authentication (2FA),
+users must configure it after their first login within 48 hours.
```

Comment:
Actually it's whatever `passwordExpirationAfterResetHours` is set to which happens to default to 48 hours right? 

I think 48 hours reads more easily but I think referring to the setting that contains the actual value is a bit more specific and correct.